cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -1,6 +1,7 @@
 package cmdtree
 
 import (
+	"github.com/ActiveState/cli/cmd/state/internal/cmdtree/internal/intercepts/cmdcall"
```

Comment:
I think you may have done something wrong here. 2 internals?

Reply:
The reason for this is because there's two important scopes here - the command call event handling, and the construction of the command call event intercept function. Keeping this in a separate package is my preference to help keep tabs on dependencies/isolation. My second preference is to have the type in the cmdtree package since that's the only place that will use the intercepts.

My third preference (with a significant preference against) is to have it in the internal/events/cmdcall package. Having the intercept there would be like having a captain command in a runner.

Reply:
Just put it in `"github.com/ActiveState/cli/cmd/state/internal/cmdtree/intercepts/cmdcall"`

The double internal is the issue.